src/commands/infra/generate.ts

```diff
@@ -576,6 +630,36 @@ export const generateConfig = async (
       templatePath
     );
   }
+  // TODO: Function to check for relative paths:
```

Comment:
by adding this block of code into this `generateConfig` function shall make the function huge and you will find it hard to write unit test. I guess you move this block of code to a separate function

Reply:
Are you referring to lines 630 - 650? Im good with that -You're saying decouple from generateConfig and place the local path manipulation in a separate function.

Reply:
yes: 630 - 654 to be precise. you can have these code in a new function and have `generateConfig` function to call this new function. The objective is to have smaller testable functions. thanks

Reply:
Updated!